receiver/dockerstatsreceiver/README.md

```diff
@@ -74,6 +74,54 @@ receivers:
 The full list of settings exposed for this receiver are documented in [config.go](./config.go)
 with detailed sample configurations in [testdata/config.yaml](./testdata/config.yaml).
 
+## Docker Socket Permissions
+
+### Requirements
+
+This receiver requires access to the Docker daemon socket to query container statistics. The Docker socket requires specific permissions:
+
+- **Linux** (`/var/run/docker.sock`): Accessible by `root` user or members of the `docker` group
+- **Windows** (`npipe:////./pipe/docker_engine`): Requires appropriate named pipe permissions
+
+Since version 0.40.0, official OpenTelemetry Collector images run as a non-root user for security. This creates a permission conflict when accessing the Docker socket.
+
+### Permission Solutions
+
+#### Grant Docker Group Access
+
+On Linux, grant the collector process access to the `docker` group:
+
+```bash
+# Get the docker group ID on your host
+getent group docker
+
+# Linux example - Run with docker group (replace 999 with your docker group GID)
+docker run --group-add 999 \
+  -v /var/run/docker.sock:/var/run/docker.sock \
+  otel/opentelemetry-collector-contrib:latest
+```
+
+#### Run as Root
+
+If running as root is acceptable for your environment:
+
+```bash
+# Linux example
+docker run -u 0 -v /var/run/docker.sock:/var/run/docker.sock \
+  otel/opentelemetry-collector-contrib:latest
+```
+
+**Note:** Running as root is not recommended for production environments.
+
+#### Alternative Approaches
+
+For enhanced security, consider:
+- Using a Docker API proxy that restricts access to only required endpoints
+- Running this receiver in an isolated collector instance with elevated privileges, forwarding metrics to your main collector via OTLP
```

Comment:
IMO, a container with elevated privileges should not expose receiver ports (e.g. OTLP, zipkin ports) as the risk of RCE is higher when ports are exposed (thinking log4J type vulnerability). If a container is elevated to collect host metrics, best practise IMO would be to have a separate container that receives push.